<launch>
 
  <!-- <node pkg="peac" type="peac_ros_node" name="peac_plane_seg" output="screen">
    <remap from="/peac_cloud_rotation/cloud_xyzrgba"   to="/read_pub_cloud/reader_cloud"/>
  </node> -->
  <node pkg="stair_modeling" type="stair_modeling_node" name="stair_modeling_node" output="screen">
    <remap from="/stair_modeling_node/vec_planes"   to="/demo_peac/vecPlane"/>
  </node>
  <!-- <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="param_setting" /> -->
  <node pkg="image_view" type="image_view" name="image_view" args="image:=/peac_plane_seg/seg_image"/>
</launch>